src/handlers/no_merges.rs

```diff
@@ -24,6 +24,8 @@ pub(super) struct NoMergesInput {
 struct NoMergesState {
     /// Hashes of merge commits that have already been mentioned by triagebot in a comment.
     mentioned_merge_commits: HashSet<String>,
+    /// Labels that the bot added as part of the no-merges check.
+    added_labels: Vec<String>,
```

Comment:
I believe this will cause a problem with any entries already in the database, as it will fail to deserialize, since this field is now required.

(And unfortunately it causes a panic, which will break triagebot handling for that pr.)

Reply:
Would using `Option<Vec<String>>` avoid that?

Reply:
I think `#[serde(default)]` should work.